maxim/logger/livekit/gemini/gemini_realtime_session.py

```
@@ -259,27 +304,60 @@ def handle_usage_metadata(self: RealtimeSession, usage: UsageMetadata):
-    turn = session_info["current_turn"]
+    turn = session_info.current_turn
     if turn is None:
         return
-    if not args or len(args) == 0:
-        return
-    audio_frame: AudioFrame = args[0]
+
     # This will help us skip silence before the turn starts
-    if len(audio_frame.data) == 0 and len(turn["turn_input_audio_buffer"]) == 0:
+    if (
+        audio_frame._data is None
```

Pylint (3.3.7)

[warning] Access to a protected member _data of a client class

(W0212)